.github/workflows/build.yml

```diff
@@ -12,5 +14,14 @@ jobs:
         with:
           java-version: '8'
           distribution: 'zulu'
+            #After decoding the secret key, place the file in ~ /. Gradle/ secring.gpg
+      - name: Decode secret key
+        run: |
+            echo "${{secrets.SIGNING_KEY_FILE}}" > ~/secretKey.gpg.b64
+            base64 -d ~/secretKey.gpg.b64 > ${{ env.SIGNING_KEY_FILE_PATH }}
       - name: Assemble
         run: ./gradlew --stacktrace assemble
+        env:
+          SIGNING_KEY_ID: ${{ secrets.SIGNING_KEY_ID }}
```

Comment:
random q: where do we go to manage secrets in GH actions?

Reply:
you need to have admin access to the repo, then go to Settings > Secrets.

Reply:
👍  TIL